tasks/src/question-answering/data.ts

```diff
@@ -43,25 +43,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["question-answering"],
 	metrics:   [
 		{
-			description: "Exact Match is a metric that is based on the strict character match of the predicted answer and the ground truth. For correct answers, EM is 1. Even if one character is different, EM will be 0.",
+			description: "The Exact Match metric is based on the strict character match of the predicted answer and the right answer. For correct predicted answers, the Exact Match will be 1. Even if only one character is different, the Exact Match will be 0.",
```

Comment:
I'd rather have it like this:
Exact Match is a metric based on the strict character match of the predicted answer and the right answer. For answers predicted correctly, the Exact Match will be 1. Even if only one character is different, Exact Match will be 0.

Reply:
Will apply :)